backstop/tests/iModel-loader.html

```diff
@@ -0,0 +1,409 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>iModel loader | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      #imodel-loader-page {
+        height: 90vh;
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+      }
+
+      svg {
+        width: 250px;
+        margin-bottom: 20px;
+      }
+    </style>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <section>
+      <div id="imodel-loader-page">
+        <!-- Use iModel-loader.svg from @itwin/itwinui-illustrations package. -->
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 740 854.43"
+        >
+          <g>
+            <polygon
+              data-id="outer3"
+              fill="#62bb47"
+              points="369.99 0 369.99 213.61 277.49 160.21 185 106.81 369.94 0.03 369.99 0"
+            >
+              <animate
+                attributeName="opacity"
+                dur="1s"
+                values="0.3;1;0.65;0.3"
+                repeatCount="indefinite"
+                begin="-0.17"
+              />
+            </polygon>
+            <polygon
+              data-id="outer2"
```

Comment:
So the only thing left, if we should consider naming them more unique? at lest `iui` prefix.. apart from that, I think this pr is good to go.
@mayank99

Reply:
This svg is already published so no point